src/components/modals/AddNewTimeSheet/AddNewTimeSheet.module.css

```diff
@@ -0,0 +1,191 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.7);
+}
+
+.modal_time {
+    background-color: var(--dark-background-color);
+    min-width: 615px;
+    min-height: 848px;
+    max-width: 615px;
+    max-height: 848px;
+    border-radius: 10px;
+    z-index: 100;
+}
+
+.modal_time_header {
+    position: relative;
+    padding: 43px 0 44px 64px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 37px;
+    color: #ffffff;
+}
+
+.mid {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+#date {
```

Comment:
It is not the best practice to style elements using id selectors, because of the specificity it will be hard to override it
Please use class selectors for applying styles

Reply:
Changes commited!